docs/marketplace/user/features/202106.0/merchant-products.md

```diff
@@ -0,0 +1,34 @@
+Merchant Products are the products that a Merchant owns. Besides creating the offers for the products that the Marketplace Administrator suggests, a Merchant can create their own unique products. These products possess the same characteristics the usual abstract and concrete products have, but in addition, every such product has Merchant-related information such as merchant reference. Merchants manage stock, set prices 
+
+## Unique and shared merchant products
+
+Merchant products can be unique or shared between several merchants. In case the product is shared, it has the owner merchant and other merchants are able to create offers for this product in the Merchant Portal.
```

Comment:
shared between several merchants can be misleading, it's more about allowing to create offers or not (think of operator /main merchant case that only allows offers in their marketplace)

Reply:
good point, thanks, will update the documentation